docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -90,15 +90,17 @@ Read more about Clerk session tokens and how they work in [the guide on how Cler
   </Tab>
 </Tabs>
 
+## Custom claims
+
 If you would like to add custom claims to your session token, you can [customize it](/docs/backend-requests/custom-session-token).
 
 You can also create custom tokens using a [JWT template](/docs/backend-requests/jwt-templates).
 
 ## Size limitations
 
-The Clerk session token is stored in a cookie. All modern browsers [limit the maximum size of a cookie to 4kb](https://datatracker.ietf.org/doc/html/rfc2109#section-6.3). Exceeding this limit can have adverse effects, including a possible infinite redirect loop for users who exceed this size in Next.js applications.
+The Clerk session token is stored in a cookie. All modern browsers [limit the maximum size of a cookie to **4KB**](https://datatracker.ietf.org/doc/html/rfc2109#section-6.3). Exceeding this limit can have adverse effects, such as possible infinite redirect loops in Next.js applications.
```

Comment:
Exceeding this limit will cause the cookie not to be set, which will guaranteed break your app since Clerk relies on cookies to function.